### PR TITLE
-XX:+EnsureHashed:java/lang/Class,java/lang/Thread added to the default options list

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
https://github.com/eclipse-openj9/openj9-docs/issues/989

Updated the related topics with this addition to the default options.

Signed-off-by: Sreekala Gopakumar <sreekala.gopakumar@ibm.com>

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have updated the documents. Please let me know your review feedback. Thanks!

**Comment by pshipton** [comment]
The changes look good, but there is a merge conflict to be resolved before this can be merged.

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - Some additional changes were made. Could you please review and let me know your feedback? Thanks!

File: docs/version0.35.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ The following new features and notable changes since version 0.33.1 are included
 - [New `-XX:[+|-]ShowNativeStackSymbols` option added](#new--xx-shownativestacksymbols-option-added)
 - [New `user2` event added for the `-Xdump` option](#new-user2-event-added-for-the--xdump-option)
 - [New `-XX:[+|-]PerfTool` option added](#new--xx-perftool-option-added)
+- [`-XX:+EnsureHashed:java/lang/Class,java/lang/Thread` is added in the `options.default` file](#xxensurehashedjavalangclassjavalangthread-is-added-in-the-optionsdefault-file)
```
Comment (pshipton) [inline docs/version0.35.md:34]: This link doesn't work in the generated doc. The previous three don't work either.

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have corrected the links.Please check and confirm. Thanks!

**Comment by pshipton** [comment]
Jenkins doc stage

**Pull request merged**